metadat/no-intro/Nintendo - Nintendo 64.dat

```diff
@@ -66,15 +66,15 @@ game (
 )
 
 game (
-	name "40 Winks (Europe) (En,Es,It) (Proto) (1999-10-07)"
-	description "40 Winks (Europe) (En,Es,It) (Proto) (1999-10-07)"
-	rom ( name "40 Winks (Europe) (En,Es,It) (Proto) (1999-10-07).z64" size 33554432 crc 7E718143 md5 0A5ACFEA0C7CF68AE25202040D5AD1EB sha1 3774E87AA383220060D330314F2C5BBB872F72CE )
+	name "40 Winks (Europe) (En,Es,It) (Proto)"
```

Comment:
40 Winks has the year at the end in No-Intro:
https://datomatic.no-intro.org/?page=show_record&s=24&n=0008